include/aspect/material_model/melt_simple.h

```diff
@@ -171,15 +173,17 @@ namespace aspect
         // entropy change upon melting
         double peridotite_melting_entropy_change;
 
+        MeltModel::MeltSimpleFraction<dim> melt_simple_fraction;
+
         /**
          * Percentage of material that is molten for a given @p temperature and
          * @p pressure (assuming equilibrium conditions). Melting model after Katz,
          * 2003, for dry peridotite.
          */
-        virtual
-        double
-        melt_fraction (const double temperature,
-                       const double pressure) const;
+        // virtual
+        // double
+        // melt_fraction (const double temperature,
+        //                const double pressure) const;
```

Comment:
Is the original code?

Reply:
This is the original code I left in there just to make sure I wasn't messing anything up during this change. I can get rid of it now!